package.json

```diff
@@ -58,6 +58,7 @@
     "@feathersjs/errors": "^4.5.11",
     "debug": "^4.3.1",
     "qs": "^6.10.1",
+    "ssri": "^8.0.1",
```

Comment:
Where did you use the module `ssri` can't find any usage.

Reply:
i haven't researched where it come from it was mentioned by dependabot https://github.com/sajov/feathers-solr/security/dependabot

Reply:
I've seen it is a sub dependency of dtslint.